packages/zk-eddsa-ticket-pcd/test/ZKEdDSATicketPCD.spec.ts

```diff
@@ -0,0 +1,218 @@
+import { EdDSAPCDPackage } from "@pcd/eddsa-pcd";
+import { EdDSATicketPCDPackage, ITicketData } from "@pcd/eddsa-ticket-pcd";
+import { ArgumentTypeName } from "@pcd/pcd-types";
+import {
+  SemaphoreIdentityPCDPackage,
+  SemaphoreIdentityPCDTypeName
+} from "@pcd/semaphore-identity-pcd";
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="../src/util/declarations/circomlibjs.d.ts" />
+/// <reference path="../src/util/declarations/snarkjs.d.ts" />
+import { Identity } from "@semaphore-protocol/identity";
+import { expect } from "chai";
+import "mocha";
+import * as path from "path";
+import { v4 as uuid } from "uuid";
+import {
+  EdDSATicketFieldsRequest,
+  ZKEdDSATicketPCD,
+  ZKEdDSATicketPCDArgs,
+  ZKEdDSATicketPCDPackage,
+  ZKEdDSATicketPCDTypeName
+} from "../src";
+
+const zkeyFilePath = path.join(
+  __dirname,
+  `../artifacts-unsafe/eddsaTicket.zkey`
```

Comment:
what is the distinction between `unsafe` and not unsafe? when will we get a not unsafe version?

Reply:
added a README here. tl;dr we haven't run a trusted setup.